WearTilesKotlin/app/src/main/java/com/example/wear/tiles/SingleEntryTimeline.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.wear.tiles
+
+import androidx.wear.tiles.LayoutElementBuilders
+import androidx.wear.tiles.LayoutElementBuilders.LayoutElement
+import androidx.wear.tiles.TimelineBuilders
+
+/**
+ * Provides a [TimelineBuilders.Timeline] which has only one entry.
+ */
+internal fun singleEntryTimeline(rootLayout: LayoutElement): TimelineBuilders.Timeline {
```

Comment:
potential candidate for horologist-tiles?

Reply:
is this a DSL? 🤔

Reply:
+ Do you mean in samples or Horologist?

Reply:
my mistake - I had the `{}` in MessagesRenderer but forgot to update that one bit!